eth/backend.go

```diff
@@ -550,6 +553,7 @@ func (s *Ethereum) Protocols() []p2p.Protocol {
 	}
 	// diff protocol can still open without snap protocol
 	protos = append(protos, diff.MakeProtocols((*diffHandler)(s.handler), s.snapDialCandidates)...)
+	protos = append(protos, trust.MakeProtocols((*trustHandler)(s.handler), s.snapDialCandidates)...)
```

Comment:
I made a mistake when implement diff sync, I think need a  `tustDialCandidates` here

Reply:
You are right, I will modify it.